### PR TITLE
Added sharing icons

### PR DESCRIPTION
**Pull request by neynah**
@dwrensha

### CODE REVIEW

File: icons/copy.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 40 40" style="enable-background:new 0 0 40 40;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:none;}
+	.st1{fill:#9E9E9E;}
```
Comment (zarvox) [inline icons/copy.svg]: Can you make these be fill: #000000 instead?  We have some build scripts that assume these icons are black, and build both a dark-background and light-background version of the icons.

(I adjusted the ones from yesterday already in https://github.com/sandstorm-io/sandstorm/pull/684 )

File: icons/email.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 40 40" style="enable-background:new 0 0 40 40;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#939393;}
```
Comment (zarvox) [inline icons/email.svg]: ditto here

File: icons/link.svg [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 40 40" style="enable-background:new 0 0 40 40;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#939393;}
```
Comment (zarvox) [inline icons/link.svg]: and here

File: icons/people.svg [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 40 40" style="enable-background:new 0 0 40 40;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#9E9E9E;}
```
Comment (zarvox) [inline icons/people.svg]: and here

**Comment by zarvox** [comment]
Consider also adding these to the Makefile, so they'll be included in the build process and the resulting bundle.

**Comment by dwrensha** [comment]
Looks good to me.

**Comment by dwrensha** [comment]
Hm. Looks like these don't have the magic "#000000" string in them that our Makefile expects for changing the fill color. I'll go ahead and add `style="fill:#000000;"` attributes where needed.

I wonder whether there's a saner way for us to be changing the fill color...

**Comment by neynah** [comment]
@dwrensha Yeah, I looked into this yesterday but apparently when I export to svg, it only adds the fill if the colour is anything but black.

Until we can figure out some other solution, I'll be sure to add the style fill to the files before I do the pull request.

**Comment by kentonv** [comment]
We can change the build scripts to look for some other color if that helps. The only important thing is that they have one specific color to look for.

**Comment by neynah** [comment]
That sounds ideal. How about #111111?

**Pull request merged**